ocaml/idl/datamodel_host.ml

```diff
@@ -1727,6 +1727,23 @@ let copy_primary_host_certs =
       ]
     ~allowed_roles:_R_POOL_ADMIN ~hide_from_docs:true ~pool_internal:true ()
 
+let set_https_only =
+  call ~name:"set_https_only"
+    ~doc:
+      "updates the host firewall to open or close port 80 depending on the \
+       value"
+    ~lifecycle:[]
+    ~params:
+      [
+        (Ref _host, "self", "The Host")
+      ; ( Bool
+        , "value"
+        , "Boolean for https_only; true - http port 80 will be blocked, false \
+           - http port 80 will be open"
```

Comment:
I would remove the first sentence altogether, that information is already displayed elsewhere (it's a boolean by the type and the value controls by its name)

```suggestion
        , "true - blocks http port 80; false - opens http port 80"
```